dev-python/regex/regex-2019.12.20.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="All things regular expression: implemented in python"
+DESCRIPTION="Regular expression module, to replace re"
```

Comment:
Maybe a better description?
This regex implementation is backwards-compatible with the standard ‘re’ module, but offers additional functionality.

Reply:
Just caught my eye when opening in HD :)

Reply:
@korli I can't push to remote branches here (at least never did), could you push some change to the description, other then that the PR is OK for me